src/hooks/useFieldValidation.tsx

```
@@ -0,0 +1,78 @@
+          );
+
+      return {
+        ...prevValue,
+        [inputName]: newArray,
+      };
+    });
+  };
+
+  useEffect(() => {
+    onComplete({
+      isComplete: isComplete && !Boolean(errorMessage),
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)